src/VisualStudio/IntegrationTest/IntegrationTests/Workspace/WorkspacesNetCore.cs

```diff
@@ -40,7 +40,7 @@ public override void MetadataReference()
             VisualStudio.SolutionExplorer.EditProjectFile(project);
             VisualStudio.Editor.SetText(@"<Project Sdk=""Microsoft.NET.Sdk"">
   <PropertyGroup>
-    <TargetFramework>net461</TargetFramework>
+    <TargetFramework>net462</TargetFramework>
```

Comment:
We can't assume net462 will be installed either. The harness only requires net46:

https://github.com/dotnet/roslyn/blob/cb400a925dc235b505ee2163dc1501d3433d2a12/src/VisualStudio/IntegrationTest/IntegrationTests/SharedIntegrationHostFixture.cs#L22

Any of the following changes would be acceptable:

1. Change the test to use net46
2. Change the test to add a `<PackageReference>` for the .NET 4.6.1 targeting pack from NuGet
3. Update the harness to require .NET 4.6.1 targeting pack be installed in the test instance
4. Update the test to use net462 _and_ perform either (2) or (3) for .NET 4.6.2

I believe the approach most likely to work in the future without maintainability problems is (2).

Reply:
Update the test to use PackageRefernece instead

Reply:
Sorry I wasn't more clear. For the PackageReference, I was expecting a reference to this package:
https://www.nuget.org/packages/Microsoft.NETFramework.ReferenceAssemblies/

The package provides targeting support for .NET Framework versions in cases where the targeting pack is not installed locally. Originally this was primarily used to provide support for builds on Linux, but it also works for providing support for scenarios like we encountered here.